input/pagecontent/4-5-multi-anchor-considerations.md

```diff
@@ -16,7 +16,11 @@ For example, an [`encounter-open`](3-4-1-encounter-open.html) implies a [`patien
 
 ### Hub derives open events
 
-The hub is responsible for identifying and sending these implied open events. When distributing a received event, a hub SHALL ensure open events are also sent to subscribers, for the referenced resource types of the received event. Hubs SHOULD NOT generate and send duplicative events. 
+The hub is responsible for identifying and sending these implied open events. When distributing a received event, the Hub is responsible for generating and communicating open events for the resource types referenced by the received event. It's important that Hubs don't generate and send duplicative events. 

```

Comment:
* Should the "open" in "communicating open events" be formatted as `open`?
* Last sentence, the spec generally avoids contractions - *could* expand these (fine either way, just calling attention).